sdk/tests/invariant.test.ts

```diff
@@ -96,8 +96,18 @@ describe('invariant', async () => {
       0n
     )
 
-    await token0.approve(account, invariant.contract.address.toString(), 1000000000n)
-    await token1.approve(account, invariant.contract.address.toString(), 1000000000n)
+    await token0.approve(
```

Comment:
You should use only one instance of `PSP22` class. This class is no longer associated permanently with specific token